models/tiktok_ads.yml

```diff
@@ -447,4 +447,73 @@ models:
       - name: daily_cpm
         description: The average amount of money you've spent per 1,000 impressions.
       - name: daily_ctr
-        description: The percentage of times people saw your ad and performed a click.
\ No newline at end of file
+        description: The percentage of times people saw your ad and performed a click.
+
+  - name: tiktok_ads__campaign_country_report
+    description: Each record in this table represents the daily performance of a campaign at the country/geographic region level.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - source_relation
+            - campaign_id
+            - advertiser_id
+            - date_day
+            - currency
+            - objective_type
+            - budget_mode
+            - split_test_variable
```

Comment:
The ERD doesn't show the PK of this table. However, when inspecting the data this seemed to align with the results as well as providing a bit more wiggle room.